setup.py

```diff
@@ -105,6 +105,7 @@ def txt_read(*paths):
         "GeoNodes": ["cartopy[plotting]", ],  # required for visualization of GeoArray and GeoSurface nodes
         "BAG": ["hydroffice.bag>=0.2.10", ],  # required by BAG plugin
         "OpenDAP": ["pydap", ],  # required by OpenDAP plugin
+        "ADIOS": ["adios>=1.9.1b19", ],  # required by ADIOS plugin
```

Comment:
> sounds good, do you know how to code that?

I am not sure that I understood your question.

You can check if the setup is running on Windows (as we do here: https://github.com/HDFGroup/hdf-compass/blob/develop/hdf_compass/utils/utils.py#L24 ), then add an `if` statement that only adds the ``ADIOS`` item to the extra requirements dictionary if NOT on Windows.